ibis/expr/operations/numeric.py

```diff
@@ -148,19 +148,31 @@ class Round(Value):
     """Round a value."""
 
     arg: StrictNumeric
-    # TODO(kszucs): the default should be 0 instead of being None
-    digits: Optional[Integer] = None
+    digits: Integer
```

Comment:
Should we restrict this to a plain python int? eg known at expression construction time? I think we do. Duckdb only supports constant precisions, eg `SELECT round(1.234, x) FROM (SELECT 2 AS x) AS t;` errors. IDK about other backends, but that seems likely as well. In the `if raw_digits == 0` code below, we are making an assumption that it is not an ibis expression.

Reply:
I'm going to make this change in a subsequent PR.